macos/generate_wazuh_packages.sh

```diff
@@ -168,9 +168,14 @@ function main() {
         case "$1" in
         "-b"|"--branch")
             if [ -n "$2" ]; then
-                BRANCH_TAG="$2"
-                BUILD=yes
-                shift 2
+                if [[ `curl https://api.github.com/repos/wazuh/wazuh-api/branches` =~ "$2" ]] || [[ `curl https://api.github.com/repos/wazuh/wazuh-api/tags` =~ "$2" ]]; then
```

Comment:
This check isn't necessary. `git` automatically checks this while cloning the branch.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20